notebooks/lesson05_random-forest-deep-dive.ipynb

```diff
@@ -0,0 +1,1577 @@
+{
```

Comment:
<p>shouldn't n_estimators=1 in this case?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>no it's correct to have 10</p>